src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,29 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+const getLinkClass = ({ isActive }: { isActive: boolean }) => {
+  return classNames('navbar-item', { 'has-background-grey-lighter': isActive });
```

Comment:
```suggestion
  return classNames('navbar-item', { 
    'has-background-grey-lighter': isActive 
  });
```
it's better to write like this and it will be more readable